tolgee-cli/faq.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: faq
+title: Frequently Asked Questions
+---
+
+This page addresses common questions and misconceptions about the Tolgee CLI.
+
+## Key Management and Default Values
+
+### Q: Does the CLI detect duplicate keys with different default values?
+
+**A: No, currently the CLI does not detect this scenario.** 
+
+If you have the same key with different default values in your code:
+
+```jsx
+<T keyName="save-btn" defaultValue="Save" />
+<T keyName="save-btn" defaultValue="Save (Changed)" />
+```
+
+The CLI will not warn you about this conflict. This is a known limitation.
+
+**Recommended approach**: Use consistent default values across your codebase, or better yet, rely on the Tolgee Platform as the single source of truth for all translations.
+
+### Q: Why don't default values automatically update my base language translations?
+
+**A: This is by design to prevent conflicts between code and platform translations.**
+
+When you first sync a key, the default value becomes the base language translation. However, subsequent changes to default values in code will not update the platform translation. This prevents situations where:
+
+- A developer changes a default value in code
+- A translator or PM has already refined the wording in the platform
+- The two sources of truth conflict with each other
+
+### Q: Can I search for strings in my code if I only store keys?
```

Comment:
How can I search for strings in my code if I only use keys and not default values?